github/ci/prow/files/jobs/vm-import-operator/vm-import-operator-periodics.yaml

```diff
@@ -0,0 +1,32 @@
+periodics:
+- name: periodic-vm-import-operator-functional-tests
+  interval: 24h
+  annotations:
+    fork-per-release: "true"
+  always_run: true
+  optional: false
+  decorate: true
+  decoration_config:
+    timeout: 3h
+    grace_period: 5m
+  max_concurrency: 6
+  labels:
+    preset-dind-enabled: "true"
+    preset-docker-mirror: "true"
+    preset-shared-images: "true"
+  spec:
+    nodeSelector:
+      type: bare-metal-external
+    containers:
+      - image: gcr.io/k8s-testimages/bootstrap:v20190516-c6832d9
+        command:
+          - "/usr/local/bin/runner.sh"
+          - "/bin/sh"
+          - "-c"
+          - "export TARGET=ocp-4.3 && automation/test.sh"
```

Comment:
Since this is now a periodic, the git repo is not checked out. I think that it needs to looks similar to this one:

```
      command:
      - "/usr/local/bin/runner.sh"
      - "/bin/sh"
      - "-c"
      - "git clone https://github.com/kubevirt/vm-import-operator.git && cd kubevirt && export TARGET=okd-4.1 && automation/test.sh"
```